mock-server/prepare.js

```diff
@@ -0,0 +1,48 @@
+// This is a prepare script that is automatically run before "start" command.
```

Comment:
Why the change to JS and requirement of node?

Reply:
This is to make it cross-platform and easier to run in CI. Alpine linux doesn't have `bash` by default (it has `sh`) and Windows doesn't have any posix shell by default